components/claim/claim-card.tsx

```diff
@@ -58,12 +58,14 @@ const ClaimCard = (props: ClaimCardProps) => {
   useEffect(() => {
     if (isSuccess && writeContractSuccess) {
       showSuccessToast("Claim successful.", width);
+      reset();
```

Comment:
wasn't this merged already?

Reply:
I thought it was, it confused me too! But I pulled all the latest from develop into this branch and nothing changed
I must have set the base branch wrong :(